types/mui-datatables/mui-datatables-tests.tsx

```diff
@@ -180,4 +198,4 @@ const options: MUIDataTableOptions = {
     },
 };
 
-<MuiCustomTable title="Awesome Table" data={TableFruits} options={options} />;
+<MuiCustomTable title="Awesome Table" data={TableFruits} options={options}/>;
```

Comment:
```suggestion
<MuiCustomTable title="Awesome Table" data={TableFruits} options={options} />;
```

Reverted autoformatting